src/privileged/blurts/FirefoxMonitor.jsm

```diff
@@ -11,8 +11,9 @@ const imageDataURIs = {
 let gExtension;
 
 this.FirefoxMonitor = {
-  init(aExtension) {
+  init(aExtension, aVariation) {
     gExtension = aExtension;
+    UI_VARIANT = parseInt(aVariation);
```

Comment:
nit: Not sure if default ESLint rules pick this up, but we may want to do `parseInt(aVariation, 10)`, although, I think the risk of having an `aVariation` value of `08` and having it incorrectly parse is probably pretty slim.